cypress.config.ts

```diff
@@ -0,0 +1,7 @@
+export default {
```

Comment:
this is not meant to be here

Reply:
Why do these keep getting put into the root? I didn't open Cypress in the root.

Reply:
weird :(